jscomp/build_tests/super_errors/expected/warnings4.res.expected

```diff
@@ -9,5 +9,4 @@
   [1;33m13[0m [2m│[0m [1;33m}[0m
   14 [2m│[0m 
 
-  You forgot to handle a possible case here, for example: 
-  #second(_) | #fourth | #third
\ No newline at end of file
+  You forgot to handle a possible case here, though we don't have more information on the value.
```

Comment:
@amiralies did something go wrong rebasing on master, or what's going on with these tests?

Reply:
I didn't update test results

Reply:
I ran the tests and what you see is the current output.